docs/UserGuide.md

```diff
@@ -319,17 +329,28 @@ Examples:
 
 #### Searching for contacts: `findc`
 
-Finds contacts whose names contain any of the given keywords, or contacts who take a particular module.
+Finds contacts by their name, module they are taking or a specified task.
+
+<div markdown="block" class="alert alert-primary">
+
+:bulb: Note on searching by name or module
+* The search is case-insensitive. e.g. `findc n/hans` will match `Hans`
+* The order of the keywords does not matter. e.g. `findc n/Hans Bo` will match `Bo Hans`
+* Partial and full words will be matched. e.g. `findc m/2030S` and `findc m/CS2030S` will match contacts who are taking `CS2030S`
+* Tasks that match at least one keyword will be returned (i.e. OR search). e.g. `findc n/Hans Bo` will match 
+  `Hans Gruber` and `Bo Yang`
+
+</div>
```

Comment:
Should this be removed or link to the box from the find command (as the semantics are the same and should be considered the same)?